vmdb/app/models/ems_refresh/parsers/openshift.rb

```diff
@@ -14,16 +14,20 @@ def get_routes(inventory)
       end
 
       def get_projects(inventory)
-        process_collection(inventory["project"], :container_projects) { |n| parse_project(n) }
+        process_collection(inventory["project"], :container_projects_openshift) { |n| parse_project(n) }
       end
 
       def parse_project(project)
         new_result = parse_base_item(project)
         new_result.except!(:namespace)
-        new_result.merge!(
-            :labels       => parse_labels(project),
-            :display_name => project.displayName
-        )
+
```

Comment:
This would probably become:

    project = @data_index.fetch_path(:container_projects, :by_name, project.name)

    unless project.metadata.annotations.nil?
        project.merge!(
            :display_name => project.metadata.annotations.displayName,
            :labels       => parse_labels(project)
        )
    end

Basically here you would fetch the already stored/indexed `namespace` and you would add the additional information you get from parsing `projects`.